.travis.yml

```diff
@@ -266,24 +266,43 @@ matrix:
     - env:
         - TEST="package build"
       script:
-        - echo "Building binary packages"
-      before_deploy:
-        - git submodule update --init libvmi xen drakvuf-doppelganging
+        - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then exit 0; fi'
+        - git submodule update --init libvmi xen
         - travis_wait 60 sh package/build.sh;
+      cache:
+        directories:
+          - package/cache
+      before_deploy:
         - git config --local user.name "drakvuf-builds"
         - git config --local user.email "drakvuf-builds@tklsoftware.com"
         - export TRAVIS_TAG=${TRAVIS_TAG:-$(date +'%Y%m%d%H%M%S')-$(git log --format=%h -1)}
         - git tag $TRAVIS_TAG
-      cache:
-        directories:
-          - package/cache
+        - sudo apt-get update
+        - sudo apt-get -q -y install bison flex libjson-c-dev autoconf-archive libxen-dev
+        - export INSTALLDIR=$PWD/usr/local/
+        - export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$PWD/usr/local/lib
+        - export C_INCLUDE_PATH=$PWD/usr/local/include
+        - export CPLUS_INCLUDE_PATH=$PWD/usr/local/include
+        - export PKG_CONFIG_PATH="$PWD/usr/local/lib/pkgconfig/"
+        - export LDFLAGS="-L$PWD/usr/local/lib"
+        - export CFLAGS="-I$PWD/usr/local/include"
+        - cd libvmi
+        - autoreconf -vif
+        - ./configure --prefix=${INSTALLDIR} --disable-kvm --disable-file --disable-bareflank --disable-safety-checks
+        - make
+        - make install
+        - cd ..
+        - ./autogen.sh
+        - ./configure
+        - make dist
       deploy:
         provider: releases
         api_key:
           secure: "$GITHUB_TOKEN"
         repo: tklengyel/drakvuf-builds
         file:
           - out/**/*
+          - drakvuf*.tar.gz
```

Comment:
This artifact is already produced as `out/drakvuf*.tar.gz` (out of `Dockerfile-final`)